tools/cli/workflowBatchCommands.go

```diff
@@ -202,13 +212,38 @@ func StartBatchJob(c *cli.Context) {
 		},
 		RPS: rps,
 	}
-	wf, err := client.StartWorkflow(tcCtx, options, batcher.BatchWFTypeName, params)
+	input, err := json.Marshal(params)
+	if err != nil {
+	}
+	workflowId := uuid.NewRandom().String()
+	request := &types.StartWorkflowExecutionRequest{
+		Domain:                              domain,
+		RequestID:                           uuid.New(),
+		WorkflowID:                          workflowId,
+		ExecutionStartToCloseTimeoutSeconds: common.Int32Ptr(int32(batcher.InfiniteDuration.Seconds())),
+		TaskList:                            &types.TaskList{Name: batcher.BatcherTaskListName},
+		Memo: &types.Memo{
+			Fields: map[string][]byte{
+				"Reason": []byte(reason),
+			},
+		},
+		SearchAttributes: &types.SearchAttributes{
```

Comment:
Same as Memo, Search attributes are subject to data converter in the sdk: https://github.com/uber-go/cadence-client/blob/master/internal/internal_workflow_client.go#L1181